crates/bevy_winit/src/lib.rs

```diff
@@ -54,7 +54,7 @@ impl Plugin for WinitPlugin {
         let create_window_reader = WinitCreateWindowReader::default();
         // Note that we create a window here "early" because WASM/WebGL requires the window to exist prior to initializing
         // the renderer.
-        #[cfg(not(target_os = "android"))]
```

Comment:
Would it make sense, rather than excluding platforms where it crashes, to only enable it on platforms where it's mandatory?

Reply:
So is the actual problem here that window creation _could_ be a startup system on all platforms except wasm? And that late-init of rendering is the actual solution to this?

In which your suggestion makes complete sense, I'm just trying to understand the broader picture of the problem :)

Reply:
I'd prefer to exclude platforms where it is problematic, and it should include macos too. 
Maybe the line is introduced only for wasm/webgl, but its side effect is unknown for other platforms, and I can't test that, since I don't have windows/linux boxes.
I'll update the pr to exclude macos too and test it.